### DISCUSSION TITLE
Python problem with module

### DISCUSSION THREAD

**Discussion by Afifette** [Afifette/mdrlol#11] [Q&A] [answered]
How can i use "pystyle" module. I see "module pystyle not found" when i start my program

**Comment by W3bParserV2** [comment, marked as answer]
### Have you installed it ?

**Start an cmd and type this :**

```$ pip install pystyle```

OR

```$ pip3 install pystyle```

**Reply by Afifette** [reply]
Ok i will do that now thx!
